samples/Sample.AzureServiceBus.InMemory/Program.cs

```diff
@@ -32,9 +32,25 @@
         asb.SQLFilters = new List<KeyValuePair<string, string>>() {
             new("IsFromSampleProjectFilter","IsFromSampleProject = 'true'")
         };
-
+        
         asb.ConfigureCustomProducer<EntityCreatedForIntegration>(cfg => cfg.UseTopic("entity-created").WithSubscription());
         asb.ConfigureCustomProducer<EntityDeletedForIntegration>(cfg => cfg.UseTopic("entity-deleted").WithSubscription());
+       
+        asb.ConfigureCustomTopicConsumer("test", cfg =>
+        {
+            cfg.UseTopic("entity-created");
+            cfg.UseDefaultOptions(); // Use default options from base consumer
```

Comment:
What's the effect of not calling this explicitly? Should we advise the user to do it?